MOTIVATION.md

```diff
@@ -0,0 +1,138 @@
+available to a wide audience of researchers.
+The single-cell data type of greatest interest to most researchers is expression
+matrices, matrices that contain cell-by-gene expression values that are the
+starting point for many downstream analyses.
+The importance of this data type calls for a specialized interface that
+maximizes its utility. This specialized interface is called the DCP "matrix
+service". The matrix service is responsible for enabling investigators to access
+and analyze HCA expression data as easily as possible.
+
+# Expression Matrix Interfaces
+
+The native interface exposed by the DCP storage system, along with the quirks of
+the DCP's data organization, would make working with the matrices challenging.
+In contrast, the matrix service aims to incorporate HCA expression data into
+researchers' existing analysis practices seamlessly.
+
+As an illustration, those familiar with dataframe manipulation should be able
+to work with HCA expression data as they might work with their own local data:
+
+```python
+>>> import hca.expression
+>>> data = hca.expression.data
+>>> data
+<hca.expression.Matrix (123456989 cells, 34183 genes) int32>
+>>> filtered_data = data[(data["percent_mito"] < .05) & (data["organ"] == "kidney")]
+>>> filtered_data
+<hca.expression.Matrix (22134 cells, 34183 genes) int32>
+>>> import scanpy.api as sc
+>>> adata = filtered_data.as_anndata()
+>>> sc.pp.normalize_per_cell(adata, counts_per_cell_after=1e4)
+...
+```
+
+Analysis package developers should be able to integrate their existing work with
+HCA data:
+
+```R
+library(HumanCellAtlas)
+library(SingleCellExpression)
+
+filtered.data <- hca %>%
+    filter(percent_mito < 0.05) %>%
+    filter(organ == "kidney")
+
+sce <- as(filtered.data, "SingleCellExperiment")
+...
+```
+
+And portal and other interactive tool developers should be able to have their
+applications make requests for HCA data with minimal overhead using web best
+practices.
+
+# Challenges
+
+A number of engineering challenges need to be addressed before the interactions
+sketched above can be realized.
+
+## Scientific Utility
+
+#### Normalization and batch effect correction.
+
+The DCP combines single-cell data from many different sources. But, batch
+effects in single-cell data are particularly strong. Correct analysis of
+combined data from the HCA requires careful handling of batch and other
+normalization issues. The matrix service could prepare a matrix from multiple
+experiments that is simple and easy to use, but such a matrix may only serve to
+enable false discoveries.
+
+#### Metadata availability.
+
+Much of the engineering work around expression matrices focuses on storing and
+delivering the expression values themselves, but practical analysis of
+expression data often turns much more on metadata. An analyst needs to be able
+to access and operate on annotations of cells, genes, and experiments.
+
```

Comment:
``While much of the engineering work for expression matrices is focused on storing and delivering the expression values, practical analysis of expression data is more dependent on the quality of metadata.``